spec/syntax/function_spec.rb

```diff
@@ -17,4 +17,16 @@
         __ensure_defimpl__(protocol, for, env)
     EOF
   end
+
+  it 'detects function calls with parenthesis' do
+    expect(<<~'EOF').to include_elixir_syntax('elixirFunctionCall', 'func')
+      func()
```

Comment:
WDYT adding coverage to more cases.
The examples on the PR description brings lots of them, but the main one that I think it's missing is when the call has a parenthesis and has a `.` like `func.()`.